src/info/langs/mod.rs

```diff
@@ -19,44 +22,35 @@ pub fn get_loc_by_language_sorted(
     language_types: &[LanguageType],
     include_hidden: bool,
```

Comment:
How do we want to handle including/excluding hidden files? I checked it out and `tokei` basically forwards the configuration to a configuration for `ignore`. `ignore` [has different behavior on Windows](https://github.com/BurntSushi/ripgrep/blob/bb8601b2bafb5e68181cbbb84e6ffa4f7a72bf16/crates/ignore/src/pathutil.rs#L21-L57). As I understand it on Windows it will be hidden if the file's stats say that it's hidden, and on all operating systems it will be hidden if the filename starts with `.`.

`ignore` is much more general-purpose than our usage, as it should work with any directory, not just a repository. So I wonder if we should even handle Windows' special case. Besides adding code complexity, hiding files via Windows file attributes isn't really applicable to git repositories.

Reply:
Oh, wait, I just realized the Windows file attributes won't apply anyway if we're reading from the commit instead of the directory.